### ISSUE TITLE
JSON-Dependency-Check report does not exist

### ISSUE THREAD

**Issue by liuhe-trec** [dependency-check/dependency-check-sonar-plugin#819]
**Describe the bug**
1.I use google cloud build as CI pipeline, and SonarQube V10.0 Dependency-Check V4.0
2.I seted the both path in SonarQube dashboard like this
```
dependency-check-report.html
```
or
```
/workspace/dependency-check-report.html
```
or
```
${WORKSPACE}/dependency-check-report.json
```
all of them had the same issue
```
03:25:57.878 DEBUG: Sensors : Dependency-Check -> Analysis Warnings import -> Zero Coverage Sensor
03:25:57.879 INFO: Sensor Dependency-Check [dependencycheck]
03:25:57.879 INFO: Process Dependency-Check report
03:25:57.882 INFO: Using JSON-Reportparser
03:25:57.884 INFO: Dependency-Check JSON report does not exists. Please check property sonar.dependencyCheck.jsonReportPath:/workspace/dependency-check-report.json
03:25:57.885 INFO: JSON-Analysis skipped/aborted due to missing report file
03:25:57.889 DEBUG: JSON-Dependency-Check report does not exist.
java.io.FileNotFoundException: JSON-Dependency-Check report does not exist.
	at org.sonar.dependencycheck.report.JsonReportFile.getJsonReport(JsonReportFile.java:37)
	at org.sonar.dependencycheck.DependencyCheckSensor.parseAnalysis(DependencyCheckSensor.java:67)
	at org.sonar.dependencycheck.DependencyCheckSensor.execute(DependencyCheckSensor.java:131)
	at org.sonar.scanner.sensor.AbstractSensorWrapper.analyse(AbstractSensorWrapper.java:64)
	at org.sonar.scanner.sensor.ProjectSensorsExecutor.execute(ProjectSensorsExecutor.java:52)
	at org.sonar.scanner.scan.SpringProjectScanContainer.doAfterStart(SpringProjectScanContainer.java:371)
	at org.sonar.core.platform.SpringComponentContainer.startComponents(SpringComponentContainer.java:188)
	at org.sonar.core.platform.SpringComponentContainer.execute(SpringComponentContainer.java:167)
	at org.sonar.scanner.bootstrap.SpringGlobalContainer.doAfterStart(SpringGlobalContainer.java:137)
	at org.sonar.core.platform.SpringComponentContainer.startComponents(SpringComponentContainer.java:188)
	at org.sonar.core.platform.SpringComponentContainer.execute(SpringComponentContainer.java:167)
	at org.sonar.batch.bootstrapper.Batch.doExecute(Batch.java:72)
	at org.sonar.batch.bootstrapper.Batch.execute(Batch.java:66)
	at org.sonarsource.scanner.api.internal.batch.BatchIsolatedLauncher.execute(BatchIsolatedLauncher.java:46)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.sonarsource.scanner.api.internal.IsolatedLauncherProxy.invoke(IsolatedLauncherProxy.java:60)
	at com.sun.proxy.$Proxy0.execute(Unknown Source)
	at org.sonarsource.scanner.api.EmbeddedScanner.doExecute(EmbeddedScanner.java:189)
	at org.sonarsource.scanner.api.EmbeddedScanner.execute(EmbeddedScanner.java:138)
	at org.sonarsource.scanner.cli.Main.execute(Main.java:126)
	at org.sonarsource.scanner.cli.Main.execute(Main.java:81)
	at org.sonarsource.scanner.cli.Main.main(Main.java:62)

03:25:57.891 INFO: Dependency-Check HTML report does not exists. Please check property sonar.dependencyCheck.htmlReportPath:/workspace/dependency-check-report.html
03:25:57.891 INFO: HTML-Dependency-Check report does not exist.
03:25:57.892 DEBUG: HTML-Dependency-Check report does not exist.
java.io.FileNotFoundException: HTML-Dependency-Check report does not exist.
	at org.sonar.dependencycheck.report.HtmlReportFile.getHtmlReport(HtmlReportFile.java:37)
	at org.sonar.dependencycheck.DependencyCheckSensor.uploadHTMLReport(DependencyCheckSensor.java:83)
	at org.sonar.dependencycheck.DependencyCheckSensor.execute(DependencyCheckSensor.java:139)
	at org.sonar.scanner.sensor.AbstractSensorWrapper.analyse(AbstractSensorWrapper.java:64)
	at org.sonar.scanner.sensor.ProjectSensorsExecutor.execute(ProjectSensorsExecutor.java:52)
	at org.sonar.scanner.scan.SpringProjectScanContainer.doAfterStart(SpringProjectScanContainer.java:371)
	at org.sonar.core.platform.SpringComponentContainer.startComponents(SpringComponentContainer.java:188)
	at org.sonar.core.platform.SpringComponentContainer.execute(SpringComponentContainer.java:167)
	at org.sonar.scanner.bootstrap.SpringGlobalContainer.doAfterStart(SpringGlobalContainer.java:137)
	at org.sonar.core.platform.SpringComponentContainer.startComponents(SpringComponentContainer.java:188)
	at org.sonar.core.platform.SpringComponentContainer.execute(SpringComponentContainer.java:167)
	at org.sonar.batch.bootstrapper.Batch.doExecute(Batch.java:72)
	at org.sonar.batch.bootstrapper.Batch.execute(Batch.java:66)
	at org.sonarsource.scanner.api.internal.batch.BatchIsolatedLauncher.execute(BatchIsolatedLauncher.java:46)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.sonarsource.scanner.api.internal.IsolatedLauncherProxy.invoke(IsolatedLauncherProxy.java:60)
	at com.sun.proxy.$Proxy0.execute(Unknown Source)
	at org.sonarsource.scanner.api.EmbeddedScanner.doExecute(EmbeddedScanner.java:189)
	at org.sonarsource.scanner.api.EmbeddedScanner.execute(EmbeddedScanner.java:138)
	at org.sonarsource.scanner.cli.Main.execute(Main.java:126)
	at org.sonarsource.scanner.cli.Main.execute(Main.java:81)
	at org.sonarsource.scanner.cli.Main.main(Main.java:62)

03:25:57.893 INFO: Process Dependency-Check report (done) | time=14ms
03:25:57.894 INFO: Sensor Dependency-Check [dependencycheck] (done) | time=15ms
03:25:57.894 INFO: Sensor Analysis Warnings import [csharp]
03:25:57.896 INFO: Sensor Analysis Warnings import [csharp] (done) | time=2ms
03:25:57.897 INFO: Sensor Zero Coverage Sensor
03:25:57.981 INFO: Sensor Zero Coverage Sensor (done) | time=84ms
03:25:57.990 INFO: SCM Publisher SCM provider for this project is: git
03:25:57.996 INFO: SCM Publisher 520 source files to be analyzed
```
and i can't see any reprots in More section in SonarQube dashboard.

so please help!

**Comment by Reamer** [comment]
Did you previously run the dependency-check plugin that creates the report?
This plugin only reads the report. The creation must be done before.

**Comment by liuhe-trec** [comment]
> 

Thank you, i think i know the reason!

**Comment by Reamer** [comment]
If you are successful, let me know so the issue can be closed.
